config/dev.exs

```diff
@@ -0,0 +1,10 @@
+import Config
+
+config :situation_room, SituationRoom.Repo,
+  database: "situation_room",
+  username: "root",
+  password: "situation",
+  hostname: "localhost",
```

Comment:
These should, ideally, be read from `System.get_env` and just set these values as a fallback/default to that call, dont hard-code these, as everyones dev machine credentials will likely be different.